__tests__/app.test.js

```diff
@@ -14,3 +24,35 @@ describe.only("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Check if returned array has three objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toHaveLength(3);
+      });
+  });
+  test("200: Returns an array of topic objects with slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.map((topic) => {
```

Comment:
What is the advantage of using a .map here? do we need the array that is returned from the map? If not then what is a better array method?
When you use an alternative method - we would still need the length check within the saem test to ensure the test itself doesn't false pass - in which case you can combine this test with the one above! :D

Reply:
- Yeah, you're right, I'm not using the returned array. I can use "forEach" to simply iterate over the array and check each element.
- I can combine both tests to ensure that the array has the correct length and that each object within it has the expected properties